book/src/tech/map/importing/README.md

```diff
@@ -1,7 +1,28 @@
 # Importing
 
 This chapter describes the process of transforming OSM extracts into A/B
-Street's map model. The steps are:
+Street's map model.
+In many cases, the quickest way to download and import pre-generated
+map data is by using the GUI, e.g. as follows:
+
+```bash
+cargo run --bin game --release # launch the game
+```
+
+Once the game has been launched, click on the Sandbox option, and then
+on the button in the top right that allows you to change map:
+
+![](https://user-images.githubusercontent.com/1825120/131584198-c58bccf0-5712-4cd6-bd62-16ba607dc4af.png)
```

Comment:
This is a nifty hosting trick, but it means the contents of this repository no longer include all of the content in the docs. I'll commit this PR, then add a followup commit to explicitly put these images in the repo too.

Reply:
> This is a nifty hosting trick, but it means the contents of this repository no longer include all of the content in the docs. I'll commit this PR, then add a followup commit to explicitly put these images in the repo too.

Entirely reasonable. I do it to minimise repo sizes but no benefit in this repo that's meant to host the docs and wants to be self standing so big :+1:

Reply:
> I do it to minimise repo sizes

And habit, a quick way to get images up there but 100% reliant on GitHub which may not be the most future proof solution to hosting images (although is pretty good).

Reply:
Avoiding file size creep is definitely important -- that's one reason I split this repo out from the code. But I don't want to one day lose the images by accident; being self-contained is safer